### PR TITLE
Use absolute path for importing components

### PR DESCRIPTION
**Pull request by anxolin**
In order to facilitate overriding ANY component, this PR replaces all the relative imports for absolute imports

Also, for the files with a lot of imports, it reorganize them a little bit grouping by type

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr1023--dexreact.review.gnosisdev.com

**Review by Velenir — APPROVED** [review]
That's a lot of changes :+1: 
But I bet nobody will use all this `./custom src` functionality

**Comment by W3stside** [comment]
> That's a lot of changes 
> But I bet nobody will use all this `./custom src` functionality

honestly, probably true

**Review by W3stside — APPROVED** [review]
yeah i trust you on this

File: src/components/TradeWidget/Price.tsx [hunk 1]
```diff
@@ -3,15 +3,19 @@ import styled from 'styled-components'
...
-import { TradeFormData } from '.'
-import { FormInputError } from './FormMessage'
-import { useNumberInput } from './useNumberInput'
+// Components
 import { OrderBookBtn } from 'components/OrderBookBtn'
 
+// TradeWidget: subcomponents
+import { TradeFormData } from 'components/TradeWidget/'
+import { FormInputError } from 'components/TradeWidget//FormMessage'
+import { useNumberInput } from 'components/TradeWidget//useNumberInput'
```
Comment (alfetopito) [inline src/components/TradeWidget/Price.tsx:17]: Too many slashes?

```suggestion
import { TradeFormData } from 'components/TradeWidget'
import { FormInputError } from 'components/TradeWidget/FormMessage'
import { useNumberInput } from 'components/TradeWidget/useNumberInput'
```

**Review by alfetopito — APPROVED** [review]
Thank you for this! Might not have all the components ever overwritten, but it's ready if anyone wants to :)

File: src/components/TradeWidget/Price.tsx (hunk 1 above)
Comment (Velenir) [reply src/components/TradeWidget/Price.tsx:17]: Not enough slashes

**Pull request merged**